ui/desktop/app/routes/scopes/scope/projects/sessions/index.js

```diff
@@ -4,88 +4,39 @@
  */
 
 import Route from '@ember/routing/route';
-import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
-import { resourceFilter } from 'core/decorators/resource-filter';
 
 export default class ScopesScopeProjectsSessionsIndexRoute extends Route {
   // =services
 
   @service session;
-  @service resourceFilterStore;
+  @service store;
 
   // =attributes
 
-  @resourceFilter({
-    allowed: ['active', 'pending', 'canceling', 'terminated'],
-    defaultValue: ['active', 'pending', 'canceling'],
-  })
-  status;
-
-  @resourceFilter({
-    allowed: (route) => route.modelFor('scopes.scope.projects'),
-    serialize: ({ id }) => id,
-    findBySerialized: ({ id }, value) => id === value,
-  })
-  project;
+  queryParams = {
+    search: {
+      refreshModel: true,
+      replace: true,
+    },
+  };
 
   // =methods
 
   /**
-   * Loads all sessions under current scope for the current user.
-   *
-   * NOTE:  previously, sessions were filtered only with API filter queries.
-   *        In an effort to offload processing from the controller, sessions
-   *        are now filtered on the client by projects and status,
-   *        while user_id filtering remains server side.
+   * Loads all sessions for the current user.
    *
    * @return {Promise{[SessionModel]}}
    */
-  async model() {
-    const { id: scope_id } = this.modelFor('scopes.scope');
+  async model({ search = '' }) {
+    const filters = {
+      user_id: { equals: this.session.data.authenticated.user_id },
```

Comment:
I don't think the daemon supports user_id yet though I haven't tested, if not, we'd have to ask the backend to add support